st2actions/conf/syslog.conf

```diff
@@ -18,15 +18,11 @@ formatter=syslogVerboseFormatter
 args=()
 
 [handler_auditHandler]
-class=st2common.log.FormatNamedFileHandler
+class=st2common.log.ConfigurableSyslogHandler
 level=AUDIT
-formatter=jsonFormatter
-args=('logs/st2actionrunner.{pid}.audit.log',)
```

Comment:
For the default logging setup (logs under `/var/log/st2/`, yes we have a separate audit log file. But we also support a logging configuration where logs get sent to syslogd. If we try to specify a file in the args to the syslog.conf configuration, it screws it up, and won't start.
If you change st2.conf to point to use syslog.conf for st2actionrunner, it won't start, unless you go and remove those lines.